README.md

```diff
@@ -1 +1,74 @@
-# catalog-of-things
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+![](https://img.shields.io/badge/Ruby-red)
+
+# Catalog of things
+
+> In this project we create a command line application to save books, music albums and games, you can add its authors, labels and genders
+and of course you can list all collections of data that is preserved using JSON files.
+
+## Screenshot
+- App UI
+
+![image](https://user-images.githubusercontent.com/58233753/153499088-16543955-3939-41eb-b16d-e201f4980543.png)
+
+- UML Diagram
+
+![image](https://user-images.githubusercontent.com/58233753/153499285-8cba7cf1-e784-4ef3-8d34-75ac3aa81eca.png)
+
+
+## Project Documentation
+
+- Use this [link](https://drive.google.com/file/d/1ICQurK07X3jPnKLJx9CTNhvXcaDh2t3t/view?usp=sharing) to watch the project documentation video.
+
+## Built With
+
+- Ruby
+
+To get a local copy up and running follow these simple example steps.
+
+Run the folloging command on your terminal <code>git clone https://github.com/AlexRS90/catalog-of-things.git</code>
+and then run the next command <code>ruby main.rb</code>
+Enjoy storing things!
+
```

Comment:
- [ ] Let's include the `Testing` section in the README with instructions on how to successfully run the tests in your project. This will help users to quickly know how to run your tests easily and this will also improve your project documentation and make your README more professional.

Reply:
hi, @julie-ify thanks for the suggestion, we'll add it to the README file